maps/map_files/USS_Almayer/USS_Almayer.dmm

```diff
@@ -42781,7 +42783,7 @@
 /obj/structure/machinery/light{
 	dir = 1
 	},
-/obj/structure/bed/chair{
+/obj/structure/bed/chair/bolted{
```

Comment:
I count six bolted changes when there's only two chairs in perma. Please log locations and changes in changelog.